src/utils/trim.rs

```diff
@@ -67,8 +75,8 @@ mod tests {
 
     #[test]
     fn test_trim_end_number() {
-        let (rest, result): (_, U256) = test_trim_end(3);
-        assert_eq!(result, (256u32.pow(2) * 30).into());
+        let (rest, result): (_, FeedId) = test_trim_end(3);
+        assert_eq!(result, REDSTONE_MARKER[6..].to_vec().into());
```

Comment:
I don't remember the goal of this test :D

Reply:
:D